src/library/Comic.types.ts

```diff
@@ -23,8 +23,10 @@ export interface Item {
   width: number;
   height: number;
   url: string;
-  fileName: string;
-  hideUntilFrame: number;
+  originalName: string;
+  originalWidth: number;
+  originalHeight: number;
+  hideUntilFrame?: number;
```

Comment:
I made `hideUntilFrame` optional because the JSON I downloaded from Driftory didn't appear to have it.